### PR TITLE
Python: content validator comparison example

### PR DESCRIPTION
**Pull request by jcarah**

### CODE REVIEW

**Comment by jcarah** [comment]
@joeldodge79 will you kindly review?

File: python/content_validator_comparison.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    for item in broken_content:
+        if item.dashboard:
+            content_type = 'dashboard'
+        else:
+            content_type = 'look'
+        id = getattr(item, content_type).id
+        name = getattr(item, content_type).title
+        errors = item.errors
+        space_id = getattr(item, content_type).space.id
+        space_name = getattr(item, content_type).space.name
```
Comment (joeldodge79) [inline python/content_validator_comparison.py]: easier to read if you do something like

```python
item_content_type = getattr(item, content_type)
id = item_content_type.id
name = item_content_type.title
<etc>
```

File: python/content_validator_comparison.py [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+        url =  '{}/{}s/{}'.format(base_url,content_type,id)
```
Comment (joeldodge79) [inline python/content_validator_comparison.py]: fun python3 thing:

```suggestion
        url =  f'{base_url}/{content_type}s/{id}'
```

File: python/content_validator_comparison.py [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+    else:
+        print('No new broken content in development branch.')
+
+
+def get_base_url():
+    """ Pull base url from looker.ini, remove port"""
+    config = configparser.ConfigParser()
+    config.read(config_file)
+    full_base_url = config.get('Looker', 'base_url')
+    base_url = full_base_url[:full_base_url.index(":19999")]
```
Comment (joeldodge79) [inline python/content_validator_comparison.py]: not documented but available:

```suggestion
    base_url = sdk.auth.settings.base_url[:full_base_url.index(":19999")]
```

**Comment by jcarah** [comment]
@joeldodge79 thanks! I have committed your recommendations. Sorry for the number of commits -- I got into a messy situation trying to amend a commit

**Pull request merged**